ehrapy/anndata_ext.py

```diff
@@ -6,6 +6,13 @@
 import numpy as np
 import pandas as pd
 from anndata import AnnData, concat
+from mudata import MuData
+from rich import print
+from rich.text import Text
+from rich.tree import Tree
+
+multi_encoding_modes = {"hash_encoding"}
+available_encodings = {"one_hot_encoding", "label_encoding", "count_encoding", *multi_encoding_modes}
```

Comment:
this was required, otherwise there are circular import errors. Might need to move this into its own "encoding_util" file?

Reply:
Or scoped imports? This cannot be here.

Reply:
Yeah, this might also help to solve the issue. Maybe we dont need the import at all. Will take a look at it.